sql-plugin/src/test/scala/com/nvidia/spark/rapids/HostAllocSuite.scala

```diff
@@ -695,14 +695,31 @@ class HostAllocSuite extends AnyFunSuite with BeforeAndAfterEach with
     class AllocOnAnotherThreadWithWarmup(override val thread: TaskThread,
         override val size: Long) extends AllocOnAnotherThread(thread, size) {
 
+      def wasteSomeTime(): Unit = {
```

Comment:
This is usually called busyWait  or spinWait  that you could use as the base for the name of the method to sound more professional.